mne/viz/tests/test_epochs.py

```diff
@@ -100,6 +100,40 @@ def test_plot_epochs():
     plt.close('all')
 
 
+def test_plot_epochs():
+    """Test epoch plotting"""
+    import matplotlib.pyplot as plt
+    epochs = _get_epochs()
+    epochs.plot(scalings=None, fig_title='Epochs', trellis=False)
+    plt.close('all')
+    epochs[0].plot(picks=[0, 2, 3], scalings=None, trellis=False)
+    plt.close('all')
+    fig = epochs.plot(trellis=False)
+    fig.canvas.key_press_event('left')
+    fig.canvas.key_press_event('right')
+    fig.canvas.key_press_event('up')
+    fig.canvas.key_press_event('down')
+    fig.canvas.key_press_event('pageup')
```

Comment:
you should test if the data remains the same if you pageup once and pagedown once